tests/testthat/test-epix_slide.R

```diff
@@ -0,0 +1,40 @@
+library(dplyr)
+library(rlang)
+
+test_that("epix_slide only works on an epi_archive",{
+  expect_error(epix_slide(data.frame(x=1)))
+})
+
+test_that("epix_slide works as intended",{
+  x <- tibble::tribble(~version, ~time_value,
+                       5, c(1:2,4),
+                       6, c(1:2,4:5),
+                       7, 2:6) %>%
+    tidyr::unnest(time_value)
+  
+  xx <- bind_cols(geo_value = rep("x",12),
+                  arrange(x,time_value,version),
+                  binary = 2^(1:12)) %>%
```

Comment:
(This example would be easier to read without running if the binary was assigned before the arrange (and then the arrange would not be needed), so we could say, for ref time 5, we see time values `c(1:2,4)`, with `binary` values `2^(1:3)`, which are filtered down to time values >= 3, so we end up only with the time_value 4 row with binary value 2^3.)